index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
```

Comment:
Please consider using HTML semantics to separate contents so it would be easier to differentiate website contents, You can read more about HTML semantic section [here](https://www.thoughtco.com/html5-section-element-3467994)